### ISSUE TITLE
Implement downloading shaders from ShaderToy

### ISSUE THREAD

**Issue by KOLANICH** [markusfisch/ShaderEditor#67]
I have tried a couple of shaders and saw them working. So I think, even though not all the shaders would work, having a feature to download and try shaders right in the app would be immediately useful.
https://www.shadertoy.com/api

**Comment by rafaellago** [comment]
I think that this may be a duplicate of #39

**Comment by KOLANICH** [comment]
Not quite, that one is mainly about compatibility, this one is mainly about  having a builtin downloader.

**Comment by markusfisch** [comment]
Having a builtin downloader for those sites would sure be cool. Will come.
